configs/fluentd-s3.yaml

```diff
@@ -0,0 +1,10 @@
+
+# This is an example configuration that allows for logs to be forwarded to an AWS S3 bucket.
+
+fluentd:  
+  s3:
+    enabled: true
```

Comment:
```suggestion
    enabled: false
```

Reply:
I know it would overwrite... but just to be safe and consistent with it.

Reply:
Thanks, missed this!

Reply:
This was to be used as an example config for enabling this feature. Should we set this as false? Should we have the example at all?

Reply:
I would say just have an example in the values.yaml file, no need for a separate file (and we should probably give a fuller example -- if just switching on enabled is enough to make it work we probably don't need this)

Reply:
I see, I didn't realize the file location